eng/build.ps1

```diff
@@ -399,12 +399,12 @@ function TestUsingRunTests() {
   $args += " --configuration $configuration"
 
   if ($testCoreClr) {
-    $args += " --tfm net6.0 --tfm net7.0 --tfm net8.0"
+    $args += " --tfm net7.0 --tfm net8.0"
```

Comment:
This is so so fragile. Save for the integration tests failing I would've missed that I needed to update the TFMS here. 

I think this is a complication we don't really need anymore. The only real distinction we need in RunTests is framework or core. Probably going to simplify RunTests to only consider that.